src/components/Pressroom/ContentsNavigation/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box, ButtonBase, Grid, Typography } from '@mui/material'
+import css from './styles.module.css'
+import { AppRoutes } from '@/config/routes'
+import NextLink from 'next/link'
+import { PRESS_LINK } from '@/config/constants'
+
+type NavItemType = {
+  label: string
+  href: string
+}
+
+export const enum PressroomIds {
+  ABOUT_US = 'about',
+  SAFE_IN_THE_NEWS = 'news',
+  PRESS_RELEASES = 'press',
+}
+
+const appendHash = (href: string) => {
+  return href.startsWith('#') ? href : `#${href}`
+}
+
+const sections: NavItemType[] = [
+  { label: 'About us', href: appendHash(PressroomIds.ABOUT_US) },
+  { label: 'Safe in the news', href: appendHash(PressroomIds.SAFE_IN_THE_NEWS) },
+  { label: 'Press releases', href: appendHash(PressroomIds.PRESS_RELEASES) },
```

Comment:
`appendHash` doesn't seem to be used for "standard" links?

```suggestion
const sections: NavItemType[] = [
  { label: 'About us', href: `#${PressroomIds.ABOUT_US} },
  { label: 'Safe in the news', href: `#${PressroomIds.SAFE_IN_THE_NEWS}` },
  { label: 'Press releases', href:`#${PressroomIds.PRESS_RELEASES}` },
```